packages/manager/modules/pci/src/projects/project/storages/databases/confirm-delete/confirm-delete.controller.js

```diff
@@ -0,0 +1,44 @@
+import get from 'lodash/get';
+
+export default class {
+  /* @ngInject */
+  constructor($translate, DatabaseService) {
+    this.$translate = $translate;
+    this.DatabaseService = DatabaseService;
+  }
+
+  $onInit() {
+    this.processing = false;
+    this.trackDatabases('table::options_menu::delete_database', 'page');
```

Comment:
Cannot be handled with a `atInternet.rename` in routing ?

Reply:
Hi @marie-j , the trackDatabases function adds the prefix 'PublicCloud::pci::projects::project::storages::databases' which is common to the whole project. Using atInternet.rename would imply having this long prefix everywhere to keep consistent, which is something we want to avoid as it will be changed soon.